mobius-extras/src/main/java/com/spotify/mobius/extras/Connectables.java

```diff
@@ -34,29 +34,72 @@ private Connectables() {
   }
 
   /**
-   * Apply a function to a {@link Connectable} in order to convert from one data type to another.
-   * This is useful for instance if you want your UI to use a subset or a transformed version of the
-   * full model used in the loop. The returned {@link Connectable} doesn't enforce a connection
+   * Convert a {@code Connectable<I, O>} to a {@code Connectable<J, O>} by applying a function from
+   * J to I for each J received, before invoking the {@code Connectable<I, O>}. This makes {@link
```

Comment:
I think you invoke the `Connection<I, O>` not the `Connectable`, right?

Reply:
Creates a `Connectable<J, O>` that transforms all `J` into `I` before forwarding it to the actual `Connection<I>`